packages/web-app-files/src/components/SideBar/Shares/Links/CreateQuickLink.vue

```diff
@@ -61,7 +61,7 @@ export default defineComponent({
     const capabilitiesRoleName = useCapabilityFilesSharingQuickLinkDefaultRole()
     const createQuickLink = () => {
       const roleName = !unref(canCreatePublicLinks)
-        ? 'none'
+        ? 'internal'
```

Comment:
Please reference this via `linkRoleInternalFolder.name` instead of a hardcoded string